doc/Language/typesystem.pod6

```diff
@@ -80,8 +80,15 @@ can keep an internal state not shared with other instances of the same class.
 The behavioural part of each object is defined in the class using I<methods>,
 while the internal state is provided via I<attributes> (variable scoped within the
 class).
-Both methods and attributes (also called members) are accessed via the C<$.> twigil or,
-in the case they are defined as I<private> via the C<$!>.
+Within the class code, attributes (also called members) can always be accessed (set or get)
+via a twigil made by the attribute sigil and a C<!> (e.g., C<$!>, C<@!>, C<%!>), even
```

Comment:
I have thought as twigil as referring to just the secondary sigil.

Talking about it this way would mean we can talk about just `.` `!` `*` `?` changing the behaviour of various variables. `«$ @ % &» X~ «. ! * ?»`